scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-1x.yaml

```diff
@@ -420,6 +420,59 @@ QFN-16-1EP_4x4mm_P0.5mm_EP2.45x2.45mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-16-1EP_4x4mm_P0.65mm_EP2.5x2.5mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'http://ww1.microchip.com/downloads/en/PackagingSpec/00000049BQ.pdf'
```

Comment:
point directly to the correct page here as well: http://ww1.microchip.com/downloads/en/PackagingSpec/00000049BQ.pdf#page=266